index.md

```diff
@@ -15,7 +15,7 @@ plagued reflection-based solutions. More details can be found in
 
 ## Where are the docs?
 
-* [User documentation](users-guide.md)
+* [User documentation](users-guide.html)
```

Comment:
Well, all links from the left-hand-side menu use `.html`; should I change them all?

(BTW, I found a few other links to `.md`, so I started by updating the PR for those, keeping the `.html` extension for now; PTAL)